recipe/meta.yaml

```diff
@@ -23,7 +23,8 @@ requirements:
     - pillow
     - pretrainedmodels ==0.7.4
     - python >=3.6
-    - timm ==0.9.2
```

Comment:
Smp 0.3.3 has a hard pin on the version of timm, see error at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=888834&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=986b1512-c876-5f92-0d81-ba851554a0a3&l=1048:

```
segmentation-models-pytorch 0.3.3 has requirement timm==0.9.2, but you have timm 0.9.16
```

Will need to update upstream at https://github.com/qubvel/segmentation_models.pytorch/blob/6db76a1106426ac5b55f39fba68168f3bccae7f8/requirements.txt#L4, and get them to make a new release before it can be changed here.

Reply:
Maintainer of https://github.com/conda-forge/pytorch-cpu-feedstock/ here. 
Thank you for pinging me @jefromson .

Sometimes it is appropriate to patch the source code.

We do this a bit with conda-forge because often we are more at the bleeding edge of releases compared to what people are working with upstream.

It may be that we should:

1. Set the pin to `>=0.9.2,<0.10a.0a` instead of `==0.9.2` with conda build's infrastructure, this translates to any "version 0.9".
2. Patch the file you pointed to with a conda-forge patch. Preferably, we leave a comment pointing to a tracking issue upstream, or even more preferred, a PR showing that the proposed patch was merged and is slated for the next release.

This is sometimes quite risky without tests. I can help try to run the small test suite in the recipe if you want.

Reply:
https://github.com/conda-forge/segmentation-models-pytorch-feedstock/pull/23 <-- leaving this here as an example of what I have in mind. but I haven't run any analysis code, so it might be fully broken.

I did leave a reference to the open issue asking for a new release with expanded `timm` compatibility.

Reply:
Keeping an eye on https://github.com/qubvel-org/segmentation_models.pytorch/pull/885 for smp to support timm 1.x